include/aos/iam/certmodule.hpp

```diff
@@ -0,0 +1,380 @@
+/*
+ * Copyright (C) 2023 Renesas Electronics Corporation.
+ * Copyright (C) 2023 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef AOS_CERTMODULE_HPP_
+#define AOS_CERTMODULE_HPP_
+
+#include "aos/common/crypto.hpp"
+#include "aos/common/time.hpp"
+#include "aos/common/tools/array.hpp"
+#include "aos/common/tools/enum.hpp"
+#include "aos/common/tools/string.hpp"
+#include "aos/iam/config.hpp"
+
+namespace aos {
+namespace iam {
+namespace certhandler {
+
+/**
+ * Module key usage name length.
+ */
+constexpr auto cModuleKeyUsageNameLen = AOS_CONFIG_CERTMODULE_KEY_USAGE_LEN;
+
+/**
+ * Maximum number of module key usages.
+ */
+constexpr auto cModuleKeyUsagesMaxCount = AOS_CONFIG_CERTMODULE_KEY_USAGE_MAX_COUNT;
+
+/**
+ * Certificate type name length.
+ */
+constexpr auto cCertificateTypeLen = AOS_CONFIG_CERTMODULE_CERT_TYPE_NAME_LEN;
+
+/**
+ * Max number of invalid certificates per module.
+ */
+constexpr auto cInvalidCertsPerModule = AOS_CONFIG_CERTHANDLER_INVALID_CERT_PER_MODULE;
```

Comment:
One const for all these items: invalid certs, invalid keys,  crtys per module should be enough. It should be the same const. And they should be moved to private CertModule modifier.